package.json

```diff
@@ -26,18 +26,18 @@
     "@babel/core": "^7.16.7",
     "@emotion/css": "^11.1.3",
     "@grafana/aws-sdk": "0.1.2",
-    "@grafana/data": "8.4.11",
-    "@grafana/e2e": "8.4.11",
+    "@grafana/data": "10.0.3",
```

Comment:
Let's make sure we test this change with 9.4 for AMG, maybe we can reach out to folks at AWS to test on their end